docs/pkg-set.8

```diff
@@ -118,29 +118,31 @@ If neither the
 .Fl g
 nor
 .Fl x
-options are used, the default is to match pkg-name exactly.
+options are used, the default is to match
+.Ar pkg-name
+exactly.
+.Sh ENVIRONMENT
+The following environment variables affect the execution of
+.Nm .
+See
+.Xr pkg.conf 5
+for further description.
+.Bl -tag -width ".Ev CASE_SENSITIVE_MATCH"
```

Comment:
```suggestion
.Bl -tag -width "CASE_SENSITIVE_MATCH"
```

Using macros in list widths is an undefined behavior, which notably renders as -1 space in some implementations.

I really love moving sections into standard order! I think it makes it more predictable!